chargeio/infrastructure/CustomDateTimeConverter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Newtonsoft.Json.Converters;
+
+namespace chargeio
+{
+    class CustomDateTimeConverter : IsoDateTimeConverter
```

Comment:
The default format used by Json.NET is the ISO 8601 standard: "2012-03-19T07:22Z".

`CustomDateTimeConverter` was created to provide the correct `DateTimeFormat` for requests to our API: "yyyy-MM-dd"